src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/refdatafixes/ReferenceDataFixesController.java

```diff
@@ -0,0 +1,87 @@
+
+    @PostMapping(value = "/updateJudgesItcoReferences", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "update the judges' ITCO references")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Accessed successfully",
+                    content = {
+                            @Content(mediaType = "application/json", schema = @Schema(implementation = CCDCallbackResponse.class))
+                    }),
+            @ApiResponse(responseCode = "400", description = "Bad Request"),
+            @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<RefDataFixesCallbackResponse> updateJudgesItcoReferences(
+            @RequestBody RefDataFixesRequest refDataFixesRequest,
+            @RequestHeader(value = "Authorization") String userToken) {
+        log.info("UPDATE JUDGES ITCO REFERENCES ---> " + LOG_MESSAGE + refDataFixesRequest.getCaseDetails().getCaseId());
+
+        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+            log.error("Invalid Token {}", userToken);
+            return ResponseEntity.status(FORBIDDEN.value()).build();
+        }
+        RefDataFixesCcdDataSource dataSource = new RefDataFixesCcdDataSource(userToken);
+        RefDataFixesData caseData = referenceDataFixesService.updateJudgesItcoReferences(
+                refDataFixesRequest.getCaseDetails(), userToken, dataSource);
+
+        return getCallbackRespEntityNoErrors(caseData);
+    }
+
+//    @PostMapping(value = "/insertClaimServedDate", consumes = APPLICATION_JSON_VALUE)
+//    @Operation(summary = "Insert the claim served date for existing cases")
+//    @ApiResponses(value = {
+//            @ApiResponse(responseCode = "200", description = "Accessed successfully",
+//                    content = {
+//                            @Content(mediaType = "application/json", schema = @Schema(implementation = CCDCallbackResponse.class))
+//                    }),
+//            @ApiResponse(responseCode = "400", description = "Bad Request"),
+//            @ApiResponse(responseCode = "500", description = "Internal Server Error")
+//    })
+//    public ResponseEntity<RefDataFixesCallbackResponse> insertClaimServedDate(
+//            @RequestBody RefDataFixesRequest refDataFixesRequest,
+//            @RequestHeader(value = "Authorization") String userToken) {
+//        log.info("INSERT CLAIM SERVED DATE ---> " + LOG_MESSAGE + refDataFixesRequest.getCaseDetails().getCaseId());
+//
+//        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+//            log.error("Invalid Token {}", userToken);
+//            return ResponseEntity.status(FORBIDDEN.value()).build();
+//        }
+//
+//        RefDataFixesData caseData = referenceDataFixesService.insertClaimServedDate(refDataFixesRequest.getCaseDetails(), userToken);
+//
+//        return getCallbackRespEntityNoErrors(caseData);
+//    }
```

Comment:
Any reason why the commented section be removed?

Reply:
It is other half of the ticket, not yet tested, so commented out